src/Surfnet/Stepup/Identity/Event/SecondFactorMigratedToEvent.php

```diff
@@ -146,8 +146,11 @@ public function obtainUserData(): array
     {
         $serializedPublicUserData = $this->serialize();
         $serializedSensitiveUserData = $this->getSensitiveData()->serialize();
```

Comment:
You could also move this to the DataFilter and check if the Event implements the Forgettable interface and then merge the sensitive data, but feel free to leave it as-is.

Reply:
That might be even better in retrospect. Making the interface more of a marker interface (save for the getter for the allowList). I will consider changing this.